server/lib/nl/utils.py

```diff
@@ -371,13 +373,34 @@ def get_sv_name(all_svs: List[str]) -> Dict:
   # Else return the name property for SV.
   for sv in all_svs:
     if sv in title_by_sv_dcid:
-      sv_name_map[sv] = title_by_sv_dcid[sv]
+      sv_name_map[sv] = clean_sv_name(title_by_sv_dcid[sv])
     else:
-      sv_name_map[sv] = uncurated_names[sv]
+      sv_name_map[sv] = clean_sv_name(uncurated_names[sv])
 
   return sv_name_map
 
 
+# TODO: Remove this hack by fixing the name in schema and config.
+def clean_sv_name(name: str) -> str:
```

Comment:
there's a similar one in https://github.com/datacommonsorg/website/blob/master/server/lib/nl/descriptions.py#L37

do you also want to add "Percentage of"?